usr.bin/du/du.c

```diff
@@ -309,36 +318,41 @@ main(int argc, char *argv[])
 			    howmany(p->fts_statp->st_blocks, cblocksize);
 
 			if (aflag || p->fts_level == 0) {
+				xo_open_instance("paths");
 				if (hflag > 0) {
-					prthumanval(curblocks);
-					(void)printf("\t%s\n", p->fts_path);
+					prthumanval("{:blocks/%4s}", curblocks);
+					xo_emit("\t{:path/%s}\n", p->fts_path);
 				} else {
-					(void)printf("%jd\t%s\n",
+					xo_emit("{:blocks/%jd}\t{:path/%s}\n",
 					    (intmax_t)howmany(curblocks *
 					    cblocksize, blocksize),
 					    p->fts_path);
 				}
+				xo_close_instance("paths");
 			}
 
 			p->fts_parent->fts_bignum += curblocks;
 		}
 		savednumber = p->fts_parent->fts_bignum;
 	}
+	xo_close_list("paths");
 
 	if (errno)
-		err(1, "fts_read");
+		xo_err(1, "fts_read");
 
 	if (cflag) {
 		if (hflag > 0) {
-			prthumanval(savednumber);
-			(void)printf("\ttotal\n");
+			prthumanval("{:total-blocks/%4s}\ttotal\n",savednumber);
 		} else {
-			(void)printf("%jd\ttotal\n", (intmax_t)howmany(
+			xo_emit("{:total-blocks/%jd}\ttotal\n", (intmax_t)howmany(
 			    savednumber * cblocksize, blocksize));
 		}
 	}
 
 	ignoreclean();
+	xo_close_container("disk-usage-information");
+	if(xo_finish() <0)
```

Comment:
Not sure why this wasn't flagged with the style checker. if space ( and there needs to be a space after <.